packages.json

```diff
@@ -3584,6 +3582,7 @@
   "https://github.com/SilverPineSoftware/UUSwiftTestCore.git",
   "https://github.com/SilverPineSoftware/UUSwiftUX.git",
   "https://github.com/simibac/ConfettiSwiftUI.git",
+  "https://github.com/simla-tech/Fastis.git",
```

Comment:
We'll need a deny list. This package is failing validation, because it's declared `swift-tools-version:5.0` but uses `.iOS(.v13)` for `platforms:`, raising:

```
ERROR: package dump failed: 'fc5a3d04-0dd7-49ef-9534-fe02f4138899': error: invalidManifestFormat("/tmp/FC5A3D04-0DD7-49EF-9534-FE02F4138899/Package.swift:30:15: error: 'v13' is unavailable
        .iOS(.v13)
              ^~~
PackageDescription.SupportedPlatform:47:27: note: 'v13' was introduced in PackageDescription 5.1
        public static let v13: PackageDescription.SupportedPlatform.IOSVersion
                          ^", diagnosticFile: nil)
```

Reply:
I'll remove it but it'll probably reappear tomorrow.